ArtOfIllusion/src/artofillusion/ArtOfIllusion.java

```diff
@@ -269,17 +269,17 @@ public static void newWindow()
     newWindow(theScene);
   }
 
-  /** Create a new window for editing the specified scene. */
-  public static void newWindow(final Scene theScene)
+  /** Create a new window for editing the specified scene and set window modified. */
+  public static void newWindow(final Scene scene, final boolean modified)
```

Comment:
Please do not make changes to the public API of major classes without discussing with the core team.